File: python/1010.py
_, units1, price1 = input().split()
units1 = int(units1)
price1 = float(price1)

_, units2, price2 = input().split()
units2 = int(units2)
price2 = float(price2)

print('VALOR A PAGAR: R$ {:.2f}'.format((units1 * price1) + (units2 * price2)))

File: c/1010.c
#include <stdio.h>

int main() {

    int units1, units2;
    double price1, price2;
    scanf("%*i %i %lf", &units1, &price1);
    scanf("%*i %i %lf", &units2, &price2);

    double total = (units1 * price1) + (units2 * price2);

    printf("VALOR A PAGAR: R$ %.2lf\n", total);

    return 0;
}
